patches/arrpc@3.5.0.patch

```diff
@@ -1,14 +1,31 @@
 diff --git a/src/process/index.js b/src/process/index.js
-index 97ea6514b54dd9c5df588c78f0397d31ab5f882a..c2bdbd6aaa5611bc6ff1d993beeb380b1f5ec575 100644
+index 389b084..d878301 100644
 --- a/src/process/index.js
 +++ b/src/process/index.js
-@@ -5,8 +5,7 @@ import fs from 'node:fs';
- import { dirname, join } from 'path';
- import { fileURLToPath } from 'url';
+@@ -1,12 +1,20 @@
+ const rgb = (r, g, b, msg) => `\x1b[38;2;${r};${g};${b}m${msg}\x1b[0m`;
+ const log = (...args) => console.log(`[${rgb(88, 101, 242, 'arRPC')} > ${rgb(237, 66, 69, 'process')}]`, ...args);
  
+-import fs from 'node:fs';
+-import { dirname, join } from 'path';
+-import { fileURLToPath } from 'url';
+-
```

Comment:
removing these imports makes the diff unnecessarily complicated. just leave them in